inference-engine/thirdparty/clDNN/src/gather_nd.cpp

```diff
@@ -0,0 +1,143 @@
+/*
+// Copyright (c) 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#include "gather_nd_inst.h"
+
+#include "primitive_type_base.h"
+#include "error_handler.h"
+#include "json_object.h"
+#include <string>
+
+namespace cldnn {
+primitive_type_id gather_nd::type_id() {
+    static primitive_type_base<gather_nd> instance;
+    return &instance;
+}
+
+static std::vector<tensor::value_type> getOrderedShape(const cldnn::tensor& ts, const cldnn::format& format) {
```

Comment:
Looks like this function does the same thing as `ts.sizes(format)`

Reply:
I applied it.